programs/spring-util/actions/blocklog.cpp

```diff
@@ -268,24 +278,26 @@ int blocklog_actions::read_log() {
 
```

Comment:
Seems we should skip over the EOS_ASSERTs for block log and the reporting if `print_from_t::fork_db` above.